README.md

```diff
@@ -195,3 +195,23 @@ In chrome browser:
 `localhost::<port>/<uri_prefix>/<file_name>`
 
 `<file_name>` is optional. It depends on the type of handler. 
+
+## Deployment
+
+`make build` to build and run webserver in docker
+
+`make deploy` to  automate the deployment
+
+This `deploy` target builds the shrunken image, copies the image to the AWS-ec2, and runs make using the new image on AWS-ec2 server. 
```

Comment:
First extracts "webserver" into the deploy directory, and then it changes directory to "deploy" and builds the shrunken image using the files in directory "deploy".